nodejs/packages/layer/src/loader.mjs

```diff
@@ -0,0 +1,71 @@
+import { register } from 'module';
+import * as path from 'path';
+import * as fs from 'fs';
+
+function _hasFolderPackageJsonTypeModule(folder) {
```

Comment:
Would it be hard to add some sort of unit/functional test?

Reply:
@tylerbenson Done